### DISCUSSION TITLE
Setting constraints on the solution as a whole

### DISCUSSION THREAD

**Discussion by luukvdmeer** [google/or-tools#3980] [Routing (and legacy CP) questions]
When solving a VRP in OR Tools it is possible to set a constraint on each route, e.g. a single route can not be longer than 10km. I am wondering if it is also possible to set such constraints on the solution as a whole, e.g. the total distance of all routes together can not be longer than 100km.

**Comment by sschnug** [comment]
See discussion in [#1595](https://github.com/google/or-tools/issues/1595) (incl. links)
